src/z3c/testing/layer.py

```diff
@@ -43,7 +40,7 @@ class BufferedDatabaseTestLayer(object):
     __bases__ = ()
     path = None
 
-    def __init__(self, config_file=None, module=__module__,
+    def __init__(self, config_file=None, module=__module__,  # noqa
```

Comment:
It surpasses: `__module__ is not defined.` I am not sure if this code ever was used successfully.